src/agent/rustjail/src/container.rs

```diff
@@ -239,7 +243,7 @@ pub struct LinuxContainer {
     pub id: String,
     pub root: String,
     pub config: Config,
-    pub cgroup_manager: Option<FsManager>,
+    pub cgroup_manager: Option<Box<dyn Manager + Send + Sync>>,
```

Comment:
Unrelated to this PR, but I wonder why wouldn't we always have a cgroup manager for a container. IOW, can we drop the `Option` here? It will simplify the code a lot by not having to check for NONE everywhere cgroup_manager is used.